recipes/radical.saga/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "radical.saga" %}
+{% set version = "0.60.0" %}
+{% set sha256 = "66045355acb70b287f63d439a929ef61c1404d83049640fbd9e0170e8e36f9f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [py3k or win]
+
+requirements:
+  host:
+    - python
```

If you accept the `noarch` suggestion you can keep the Python constraint using:
```suggestion
    - python <3
```